utils/config.env

```diff
@@ -41,10 +41,10 @@ KAFKA_BRANCH=2.4
 #   REPOSITORY=<account>.dkr.ecr.us-west-2.amazonaws.com/confluentinc   # confluentinc repository at the specified ECR registry, images must be pulled separately
 #####################################################
 
-CP_VERSION_MAJOR=5.5
-CP_VERSION_FULL=5.5.0
-GIT_BRANCH=master
-JAR_VERSION=5.5.0-SNAPSHOT
+CP_VERSION_MAJOR="$CONFLUENT_MAJOR.$CONFLUENT_MINOR"
+CP_VERSION_FULL="$CONFLUENT_MAJOR.$CONFLUENT_MINOR.$CONFLUENT_PATH"
+GIT_BRANCH=$CONFLUENT_RELEASE_TAG_OR_BRANCH
```

Comment:
Why is `GIT_BRANCH` necessary if there is already `CONFLUENT_RELEASE_TAG_OR_BRANCH`?

If `GIT_BRANCH` is used elsewhere in the repo, it should be replaced with `CONFLUENT_RELEASE_TAG_OR_BRANCH`.

Reply:
I really prefer to make minimum possible changes per commit / PR.   The associated JIRA is also very specific and not related to the projects (music & microservices) that depend on GIT_BRANCH and JAR_VERSION, so I would prefer to leave those to other commits.   Small PRs are easier to find issues in, like you did above.

Reply:
I've added DEVX-1641 to track this